hera_mc/mc_session.py

```diff
@@ -3635,8 +3641,39 @@ def add_ant_metric(self, obsid, ant, pol, metric, val):
 
         self.add(AntMetrics.create(obsid, ant, pol, metric, db_time, val))
 
+    def _metric_time_to_time_filter(self, starttime, stoptime):
+        """
+        Aligns time keywords between metric functions and _time_filter.
+
+        Convert times passed into antenna or array metrics to the format and
+        default used in _time_filter.
+
+        Parameters
+        ----------
+        starttime : astropy Time object OR gps second.
+            starttime keyword to be converted from metric input to _time_filter input
+        stoptime : astropy Time object OR gps second.
+            starttime keyword to be converted from metric input to _time_filter input
+
+        Returns
+        -------
+        starttime : astropy Time object
+            The starttime to be passed to _time_filter.
+        stoptime : astropy Time object
+            The stoptime to be passed to _time_filter.
+        """
+        if starttime is None:
+            starttime = Time(0, format='gps')
+        elif not isinstance(starttime, Time):
+            starttime = Time(starttime, format='gps')
+        if stoptime is None:
+            stoptime = Time(0, format='gps')
+        elif not isinstance(starttime, Time):
+            stoptime = Time(stoptime, format='gps')
+        return starttime, stoptime
```

Comment:
Thanks, I'll get working on finishing it up.

The default is to match our existing default (to not break workflows), but also to match the behavior I would expect from a typical db query. The specific value of 0 is just something long before any of these data existed.

Reply:
I guess maybe we differ on what we expect from a db query (and this may be because real time status info is slightly different than the kind of metrics you're working with). But the most common question by far that I see is: what is the most recent information? Which is why that's the default for the other methods.

A question about current workflows: what workflows are currently querying this information?